### PR TITLE
Room removal

### PR DESCRIPTION
**Pull request by fiwitek**

### CODE REVIEW

File: src/main/java/com/ustudent/resquod/controller/RoomController.java [hunk 1]
```diff
@@ -81,6 +81,20 @@ public ResponseTransfer changeRoomData(
         return new ResponseTransfer("Successfully updated!");
     }
 
-
-}
-
+    @ApiOperation(value = "Remove Room", authorizations = {@Authorization(value = "authkey")})
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Room Removed Succesfully"),
+            @ApiResponse(code = 400, message = "Permission Denied"),
```
Comment (ziorkie) [inline src/main/java/com/ustudent/resquod/controller/RoomController.java]: 401 code for permission denied

File: src/main/java/com/ustudent/resquod/controller/RoomController.java [hunk 2]
```diff
@@ -83,7 +79,7 @@ public ResponseTransfer changeRoomData(
 
     @ApiOperation(value = "Remove Room", authorizations = {@Authorization(value = "authkey")})
     @ApiResponses(value = { @ApiResponse(code = 200, message = "Room Removed Succesfully"),
-            @ApiResponse(code = 400, message = "Permission Denied"),
+            @ApiResponse(code = 400, message = "\"Permission Denied\" or \"Invalid Input\""),
```
Comment (ziorkie) [inline src/main/java/com/ustudent/resquod/controller/RoomController.java:82]: 401 code for permission denied

File: src/main/java/com/ustudent/resquod/service/RoomService.java [hunk 3]
```diff
@@ -76,7 +79,8 @@ public void removeRoom(NewRoomData roomToRemove) {
                 (admin.getRole().equals("ROLE_OWNER") && admin.getCorporations().contains(corporation))))
             throw new PermissionDeniedException();
 
-        roomRepository.removeRoomById(roomToRemove.getId());
+        roomRepository.delete(room);
+        //roomRepository.removeRoomById(room.getId());
```
Comment (ziorkie) [inline src/main/java/com/ustudent/resquod/service/RoomService.java:83]: unnecessary code

File: src/main/java/com/ustudent/resquod/controller/RoomController.java (hunk 2 above)
Comment (dzienki) [reply src/main/java/com/ustudent/resquod/controller/RoomController.java:82]: 401 unauthorized
403 forbidden

**Pull request closed without merging**